token-validation-spring-test/src/main/java/no/nav/security/token/support/spring/test/MockLoginController.java

```diff
@@ -60,4 +59,41 @@ public Cookie addCookie(
         }
         return cookie;
     }
+
+    @Unprotected
+    @PostMapping("/cookie")
+    public Cookie addCoaaokie(
```

Comment:
`addCoaaikie` should probably be `addCookie`

Reply:
Yep 🙊